tools/vulnerability-scan/wss-scan.sh

```diff
@@ -31,14 +31,16 @@ fi
 if [ ! -f "wss-unified-agent.jar" ]
 then
   # Download the WhiteSource Agent 
-  wget -q https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
-  # The version 20.9.2.1 has been tested and can be used if a specific version is required
-  #wget -q  https://github.com/whitesource/unified-agent-distribution/releases/download/v20.9.2.1/wss-unified-agent.jar
+  curl https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar --output wss-unified-agent.jar
+  # The version 21.11.1 has been tested and can be used if a specific version is required
```

Comment:
Can we lock to a version?  I would rather we intentional rev the version and see something break then get surprised when we start downloading a new major release

Reply:
Changed. We will now stick with unified agent v21.11.1 until further change.